Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
Yeah remember something along those lines too. Would definitely be great to have something like that. Do think it would help a lot.

Still feel it's better to have than not, it makes it quick and easy to create new content types and even guides you in terms of saying, place template x there and it will work.

+1 for improving documentation, e.g. this how to could use some improvements http://neos.readthedocs.io/en/latest/HowTos/CreatingContentElement.html
Also the documentation isn't that helpful for new comers, it's a bit hard to navigate and doesn't really have a natural flow. A beginner guide would be useful.

Probably do as you suggest, adding a custom one in this package wouldn't work well.